packages/astro/src/core/config/config.ts

```diff
@@ -78,15 +78,17 @@ export function resolveRoot(cwd?: string | URL): string {
 	return cwd ? path.resolve(cwd) : process.cwd();
 }
 
+export const configPaths = Object.freeze([
```

Comment:
It seems these paths are in order of possible appearance, i.e. `astro.config.mjs` is most likely to appear first. Can we leave a comment to say that?